tests/test_api.py

```diff
@@ -497,6 +497,30 @@ def test_comment_staff(self):
         self.assert_403(response)
         assert response.json['code'] == 403
 
+        def test_get_comments(self):
+            self._test_backup(True)
+            user = User.lookup(self.user1.email)
+            backup = Backup.query.filter(Backup.submitter_id == user.id).first()
+            comment_url = "/api/v3/backups/{}/comment/".format(encode_id(backup.id))
+
+            #check to see if student can view comments on own backup's comments
+            self.login(self.user1.email)
+            response = self.client.get(comment_url)
+            self.assert_200(response)
+            self.logout()
+
+            #check to see if staff can access comments
+            self.login(self.staff1.email)
+            response = self.client.get(comment_url)
+            self.assert_200(response)
```

Comment:
When creating a comment instance, how do I know what to put in the filename parameter? Can I, for example, assume that a file named fizzbuzz.py has been created and placed inside of the test backup? I'm looking at the test prior to this one, test_comment_staff, as an example and I can't seem to find any line that explicitly says that the test backup in that function contains a file called fizzbuzz.py.